src/getHumanAge.test.js

```diff
@@ -3,8 +3,104 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  // below are postive tests for EC and BV test designs
+  // including different values=)
+  test('check if 0, 0 inputs give 0,0 output', () => {
+    const humanAges = getHumanAge(0, 0);
+
+    expect(humanAges)
+      .toEqual([0, 0]);
+  });
+
+  test('check if 14, 14 input give 0, 0 output', () => {
```

Comment:
I would suggest:
'should return 0 for cat and dog if they are less than 15 years old'

Reply:
Agree. Fixed.